pkg/mediatype/types.go

```diff
@@ -7,54 +7,58 @@ var AVI, _ = New("video/x-msvideo", "", "avi")
 var AVIF, _ = New("image/avif", "", "avif")
 var Binary, _ = New("application/octet-stream", "", "")
 var BMP, _ = New("image/bmp", "Bitmap Image File", "bmp")
-var CBZ, _ = New("application/vnd.comicbook+zip", "Comic Book Archive", "cbz")
+var CBZ, _ = New("application/vnd.comicbook+zip", "Comic Book ZIP Archive", "cbz")
```

Comment:
I'm going to leave them in for now, because a use case for the toolkit might be to give a more user-friendly description of the files in publication in logs or messages. I do agree they are unnecessary for the code inside the toolkit itself.